doc/cephadm/upgrade.rst

```diff
@@ -48,6 +48,27 @@ The automated upgrade process follows Ceph best practices.  For example:
 Starting the upgrade
 ====================
 
+.. note::
+
+   Cephadm by default reduces `max_mds` to `1`. This can be disruptive for large
+   scale CephFS deployments because the cluster cannot easily reduce active MDS(s)
+   to `1`. Therefore, To upgrade MDS(s) without reducing `max_mds`, the
```

Comment:
```suggestion
   Cephadm by default reduces `max_mds` to `1`. This can be disruptive for large
   scale CephFS deployments because the cluster cannot quickly reduce active MDS(s)
   to `1` and a single active MDS cannot easily handle the load of all clients even for a short time. Therefore, to upgrade MDS(s) without reducing `max_mds`, the
```

Reply:
Done, thanks.